libyul/backends/wasm/polyfill/Keccak.yul

```diff
@@ -18,7 +18,332 @@
 
 // NOTE: This file is used to generate `ewasmPolyfills/Keccak.h`.
 
+// reset the context
+function keccak_reset(context_offset:i32) {
+	// state context
+	//	typedef struct {
+	//		union {                                 // state:
+	//			uint8_t b[200];                     // 8-bit bytes
+	//			uint64_t q[25];                     // 64-bit words
+	//		} st;
+	//		int pt, rsiz, mdlen;                    // these don't overflow
+	//	} sha3_ctx_t;
+	memset(i32.add(context_offset, 0:i32), 0:i32, 200:i32)
+	//	c->mdlen = mdlen;
+	//	c->rsiz = 200 - 2 * mdlen;
+	//	c->pt = 0;
+	i64.store(mdlen_offset(context_offset), 32)
```

Comment:
I guess `mdlen` is the length of the algorithm output and it's 32 because we're implementing SHA256? I think that a comment saying that would  be useful. Also we could rename `mdlen` to `digest_length`, which would make it more obvious.